client/reader/_style.scss

```diff
@@ -215,11 +215,12 @@
 
 // Custom breakpoints that drop action labels text
 // to prevent Likes text from breaking onto a new line
-.reader-share__button-label,
-.comment-button__label-status,
-.like-button__label-status {
+.is-group-reader .reader-share__button-label,
```

Comment:
@bluefuton I had to scope this to .is-group-reader because it was trumps the styles that I'm assigning to the action labels.